saleor/cart/models.py

```diff
@@ -29,13 +29,20 @@ def find_open_cart_for_user(user):
     return carts.first()
 
 
-class ProductGroup(ItemList):
+class ProductGroup(list):
```

Comment:
Not a big issue, but this wrapper could probably be replaced with two simple functions.